src/app/ApplicationTemplate.UWP/Views/Styles/Controls/CommandBar.xaml

```diff
@@ -118,8 +118,10 @@
 				Value="{ThemeResource AlternateBackgroundBrush}" />
 		<Setter Property="Foreground"
 				Value="{ThemeResource AlternateOnBackgroundBrush}" />
-		<android:Setter Property="Height"
-						Value="100" />
+		<Setter Property="Height"
```

Comment:
please add a comment saying this is a workaround and linking the github issue too